maestrowf/interfaces/script/_flux/flux0_18_0.py

```diff
@@ -11,7 +11,7 @@
 LOGGER = logging.getLogger(__name__)
 
 try:
-    import flux
+    from flux import constants, Flux, job
```

Comment:
Are job/constants maybe too generic here?  want a flux or f prefix or something on these? Those seem ripe for usage elsewhere in maestro..

Reply:
You're right -- I ended up having to go through the adapter and update local variables that aliased with these. These are local to this module though, so I don't have much concern of them bleeding out elsewhere. I can tidy these up though.

Reply:
yeah, more wondering about bleeding into this module from outside, not the other way